packages/provider/package.json

```diff
@@ -1,22 +1,24 @@
 {
-	"name": "common",
+	"name": "provider",
```

Comment:
though you were calling it "data-provider"? which is a bit more descriptive then just "provider" (as in, "what does it provide?")
https://github.com/overture-stack/lyric/pull/36/files#diff-c7ffeb6163ab1fc5929df80eda11f4999ae853d5a2bfce072cbf787c02dc0f3cR2

Reply:
it makes more sense `data-provider` will change accordingly